blocks/edit/da-editor/da-editor.js

```diff
@@ -18,9 +15,11 @@ export default class DaEditor extends LitElement {
     super.connectedCallback();
     this.shadowRoot.adoptedStyleSheets = [sheet];
     this.shadowRoot.createRange = () => document.createRange();
-    loadIms().then(() => {
-      this._imsLoaded = true;
-    });
+    if (!this._imsLoaded) {
```

Comment:
Is this necessary? Connected callback should only be called when the component is attached to the DOM... which would mean this is never true.

https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements#:~:text=called%20each%20time%20the%20element%20is%20added%20to%20the%20document.

Reply:
Good point, removed it.